dataflowengineoss/src/main/scala/io/shiftleft/dataflowengineoss/queryengine/Engine.scala

```diff
@@ -125,12 +124,45 @@ class Engine(context: EngineContext) {
     forCalls ++ forArgs
   }
 
+  private def paramToArgs(param: nodes.MethodParameterIn): List[nodes.Expression] =
```

Comment:
for more consistency: should all these defs return only one of `List|Traversal`, rather than a wild mix?